user_guide_src/source/installation/running.rst

```diff
@@ -129,6 +129,72 @@ e.g., ``apache2/conf/extra/httpd-vhost.conf``::
 If your project folder is not a subfolder of the Apache document root, then your
 <VirtualHost> element may need a nested <Directory> element to grant the web server access to the files.
 
+With mod_userdir (shared hosts)
+--------------------------------
+
+A common practice in shared hosting environments is to use the Apache module "mod_userdir" to enable per-user Virtual Hosts automatically. Additional configuration is required to allow CodeIgniter4 to be run from these per-user directories.
+
+The following assumes that the server is already configured for mod_userdir. A guide to enabling this module is available `in the Apache documentation <https://httpd.apache.org/docs/2.4/howto/public_html.html>`_.
+
+Because CodeIgniter4 requires the server to find the framework front controller at ``/public/index.php``, you must specify this location as an alternative to search for the request, even if CodeIgniter4 is installed within the per-user web directory.
```

Comment:
```suggestion
Because CodeIgniter4 expects the server to find the framework front controller at ``/public/index.php`` by default, you must specify this location as an alternative to search for the request (even if CodeIgniter4 is installed within the per-user web directory).
```